tools/milestone.py

```diff
@@ -0,0 +1,114 @@
+"""Script for automatically creating a milestone in repositories.
+
+Only specified repositories (via environment variable) will be handled.
+
+The purpose of this code is to connect to a specific repository
+and create a milestone. This milestone will be associated to a certain date
+provided as an input argument.
+
+"""
+
+import datetime
+import os
+
+import github
+
+# Insert your credentials... None by default
```

Comment:
Should we force the usage of env variables?

Reply:
Well, it depends. At the moment both input methods are supported:
* Directly modifying the script with the value
* Reading from env variable

The env variable method is intended for the GH workflow mainly. I don't see any advantage in forcing users to locally define the env vars RELEASE_DATE and REPOSITORY. Those should be kind of dynamic. In a sense that users should be able to modify them quickly. And modifying env vars is not something most people is used to. That's why I decided to put them as inputs of the script as well.

We could even modify the script to use input arguments. But maybe that's too much for what we need. It is intended to be kept simple, and we will only be using it from the workflow most probably... :S 

What do you think @jorgepiloto? Why would you force the usage of env vars? =)

Reply:
Thanks for the reply, @RobPasMue. My only concern was regarding the `MY_PAT` variable. I was thinking about avoiding plain text secrets.

 However, if this script is expected to be run thought GH and reading variables from the session environment, I think its okay.

Reply:
Yeah, I can understand your concern regarding the ``MY_PAT``. But that will only occur when you run it locally. So, no worries. When running locally you can always have that value as an env var and let the script read it =)

Thanks anyway for your suggestion!